Archon.Http.Tests/FakeHttpHandler.cs

```diff
@@ -3,21 +3,17 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Web.Http;
-using System.Web.Http.Hosting;
 
-namespace Archon.WebApi
+namespace Archon.Http.Tests
 {
-	public class FakeHttpHandler : HttpMessageHandler
+	class FakeHttpHandler : HttpMessageHandler
```

Comment:
why internal now instead of public? Are we going to stop using `FakeHttpHandler` like you were talking about?